README.md

```diff
@@ -34,17 +35,22 @@ From the repository's root directory, run:
 ```
 docker-compose up
 ```
-You should now have a web server and database server running. Opening `http://localhost:8000` in your browser will open the application.
 
 If it's your first time running the application, you'll also need to run database migrations and install Mathesar types and functions:
 ```
 docker exec mathesar_web_1 python manage.py migrate
 docker exec -it mathesar_web_1 python install.py
 ```
 
-If you want to use Mathesar with a preexisting Postgres DB, modify the `DATABASES.mathesar_tables` entry of the `config/settings.py` file with appropriate connection details before installing the Mathesar types and functions with the previous step.
+You should now have a web server and database server running. Opening `http://localhost:8000` in your browser will open the application. For sample table data, you can create a new table in the UI using the `patents.csv` file found in `/mathesar/tests/data`. 
 
-For sample table data, you can use the `patents.csv` file found in `/mathesar/tests/data`. 
+It is recommended that you keep the Docker containers running while you make changes to the code. Any change to the code made locally will sync to the container and the version deployed at `http://localhost:8000` will always be the latest local version of the code.
+
+### Configuration Options
+
+If you want to use Mathesar with a preexisting Postgres DB, modify the `DATABASES.mathesar_tables` entry of the `config/settings.py` file with appropriate connection details before installing the Mathesar types and functions by running `install.py` as described in the previous step. 
+
+**Please don't do this unless you have full confidence in what you're doing since Mathesar is not production-ready yet and may cause unexpected issues.**
```

Comment:
@kgodey This sounds like a big risk. Would we be able to trace back unexpected issues related to this easily?

Reply:
I have no idea what issues would be caused because Mathesar isn't stable yet and won't be for a couple of months. We keep breaking things, which is expected at this stage of development. I would not point it at any real production data until we release an official alpha release.

Reply:
Ok. My concern was that an inexperienced/nefarious contributor would do this and a couple months from now an issue would pop up and we wouldn't be able to trace back the problem.

Reply:
Oh, I see what you're saying. It wouldn't cause any issues to Mathesar's code or functionality, it would only (potentially) cause issues to the contributor's own database or data.

Reply:
Oh that makes sense. Thanks!

Reply:
I'll do a quick PR to clarify. Thanks for asking!